Assets/MixedRealityToolkit-SDK/Features/Boundary/Components/Scripts/PlayAreaVisualizer.cs

```diff
@@ -48,26 +39,32 @@ public bool IsPlayAreaVisualized
         private IMixedRealityBoundarySystem boundaryManager = null;
         private IMixedRealityBoundarySystem BoundaryManager => boundaryManager ?? (boundaryManager = MixedRealityManager.Instance.GetManager<IMixedRealityBoundarySystem>());
 
+        private MixedRealityBoundaryVisualizationProfile visualizationProfile;
```

Comment:
Let's lose the profile reference and make sure we're always reading the current profile from the Config